docs/conf.py

```diff
@@ -71,14 +71,17 @@
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
 #
-html_logo = "_static/watertap_reflo_logo.png"
+html_logo = "_static/WaterTAP_REFLO.png"
 
 # The name of an image file (relative to this directory) to use as a favicon of
 # the docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
 #
 html_favicon = "_static/watertap_reflo_logo.ico"
 
+html_theme_options = {
+    'style_nav_header_background': '#22577A',
+}
```

Comment:
Is anyone opposed to changing the background color where the logo is placed so it stands out better? (I'll change the color of the logo text to a lighter gray later.) Any suggestions for colors?

Reply:
@zacharybinger I think it looks good now (with your change here), but since you asked for suggestions for colors, how about setting the logo text to white?